apps/core/scenes/results/Results.js

```diff
@@ -26,12 +30,30 @@ export default class Results extends React.Component<Props> {
   };
 
   render() {
-    const { travelFrom, travelTo, dateFrom, dateTo } = this.props;
+    const {
+      travelFrom,
+      travelTo,
+      dateFrom,
+      dateTo,
+      returnDateFrom,
+      returnDateTo,
+    } = this.props;
+
+    // @TODO get from config
```

Comment:
No I didn't :) During my work I spot many things which should be handled in the future. And I didn't want to leave them forgotten. I edited description of Trip list issue, and most of the TODOs are written there. 

Should I remove TODOs from the code? I wouldn't create issues for every TODO, because it could be messy. I propose to write all of them to the main issue and remove them from the code if it's a problem to keep them also in the code.

Reply:
No, we can keep them in the code, I was just worried a lite bit 😨 😄

Reply:
Well when this PR will be merged I'm going to fix some of the TODOs. But I can't do more in this PR. Even now it's terribly huge 😢